src/System.Security.Cryptography.Xml/tests/XmlLicenseTransformTest.cs

```diff
@@ -83,6 +85,176 @@ public void OutputTypes()
             transform = new UnprotectedXmlLicenseTransform();
             Assert.NotNull(transform.OutputTypes[0]);
         }
+
+        [Fact]
+        public static void ItDecryptsLicense()
+        {
+            using (var key = RSA.Create())
+            {
+                string expected;
+                string encryptedLicenseWithGrants = GenerateLicenseXmlWithEncryptedGrants(key, out expected);
+
+                Assert.Contains("hello", expected);
+                Assert.DoesNotContain("hello", encryptedLicenseWithGrants);
+
+                XmlNamespaceManager nsManager;
+                XmlDocument toDecrypt = LoadXmlWithLicenseNs(encryptedLicenseWithGrants, out nsManager);
+
+                var decryptor = new XmlLicenseEncryptedRef();
+                var transform = new XmlLicenseTransform()
```

Comment:
Is this transform not ever used in SignedXml or EncryptedXml?

It seems like perhaps not, since it requires the extra Decryptor property-set.  Which sort of makes me wonder if it's something we should be porting.  But, I guess we've already done it, and done the work...